jupyter_scheduler/task_runner.py

```diff
@@ -260,15 +260,7 @@ def delete_job_definition(self, job_definition_id: str):
         self.cache.delete(job_definition_id)
 
     def create_job(self, job_definition_id: str):
-        definition = self.scheduler.get_job_definition(job_definition_id)
-        if definition and definition.active:
-            input_uri = self.scheduler.get_staging_paths(definition)["input"]
-            self.scheduler.create_job(
-                CreateJob(
-                    **definition.dict(exclude={"schedule", "timezone"}, exclude_none=True),
-                    input_uri=input_uri
-                )
-            )
+        self.scheduler.create_job_from_definition(job_definition_id, CreateJobFromDefinition())
```

Comment:
If we remove the is active test from `create_job_from_definition` do we need it here?

Reply:
Correct, this needs an update.

Reply:
Updated